src/profile-logic/gecko-profile-versioning.js

```diff
@@ -1316,5 +1316,32 @@ const _upgraders = {
       processes.meta.markerSchema = [];
     }
   },
+  [23]: profile => {
+    // The browsingContextID inside the pages array and activeBrowsingContextID
+    // have been renamed to tabID and activeTabID.
```

Comment:
Can you please explain here and in the other upgrader the difference between the 2 values? As well as a reference to the bugzilla bug introducing the format change.

Reply:
Good idea. Added comments that explain these.